theme/reference.css

```diff
@@ -187,6 +195,25 @@ blockquote > p {
     margin-bottom: 0px;
 }
 
+.target-specific blockquote {
+    margin-left: 1em;
+    margin-bottom: 1em;
+    margin-right: 6vw;
+    background: #00f82e;
+    padding: 0.2ex 0.5ex;
+    border-radius: 5px;
+    display: block;
+    box-shadow: inset -1px -1px 0 #06a430;
+    font-size: 0.9em;
+}
+
+.ayu  .target-specific blockquote,
+.navy .target-specific blockquote,
+.coal  .target-specific blockquote {
+    background: #008f2e;
+    box-shadow: inset -1px -1px 0 #03a430;
+}
+
```

Comment:
Maybe we can get rid of any color-based changes for now and deal with look and feel in separate PRs. I wonder if you could for now add a new column to the table with a `*` or some other marker to make the representation instead?